nlu_hyperopt/app.py

```diff
@@ -4,7 +4,16 @@
 import logging
 import sys
 
-from nlu_hyperopt.space import search_space
+input_search_space = os.environ.get("INPUT_SEARCH_SPACE")
```

Comment:
We should only do this if we are in a github action, right? So maybe check instead if the github actions typical env variables are present.

Reply:
I changed all the names to start with `INPUT_` so that the local and github action env vars are the same. Someone could use this locally too  i.e. set `INPUT_SEARCH_SPACE` to their `space.py` file without copying to the directory